todo.js

```diff
@@ -21,31 +21,32 @@ export default function Todo(props) {
               <View style={styles.flatlistView}>
                 <TouchableOpacity onPress={() => props.changeCheck(item.key)}>
                   {item.check ? (
-                    <View style={styles.touchView}>
-                      <Icon
-                        name="checkbox-marked-circle-outline"
-                        size={30}
-                        color={Colors.CornflowerBlue}
-                      />
-                      <Text style={styles.flatlistText1}> {item.text} </Text>
-                      <Entypo
-                        name="dot-single"
-                        size={30}
-                        color={Colors.green}
-                      />
-                    </View>
+                    <Icon
+                      name="checkbox-marked-circle-outline"
+                      size={30}
+                      color={Colors.CornflowerBlue}
+                    />
                   ) : (
-                    <View style={styles.touchView}>
-                      <Icon
-                        name="checkbox-blank-circle-outline"
-                        size={30}
-                        color={Colors.CornflowerBlue}
-                      />
-                      <Text style={styles.flatlistText}> {item.text} </Text>
-                      <Entypo name="dot-single" size={30} color={Colors.red} />
-                    </View>
+                    <Icon
+                      name="checkbox-blank-circle-outline"
+                      size={30}
+                      color={Colors.CornflowerBlue}
+                    />
```

Comment:
same here for icons.
see the comment below.

Reply:
name={item.check ? "checkbox-marked-circle-outline" : "checkbox-blank-circle-outline" }